pkg/util/log/eventpb/contention_events.proto

```diff
@@ -0,0 +1,33 @@
+// Copyright 2021 The Cockroach Authors.
+//
+// Use of this software is governed by the CockroachDB Software License
+// included in the /LICENSE file.
+
+syntax = "proto3";
+package cockroach.util.log.eventpb;
+option go_package = "github.com/cockroachdb/cockroach/pkg/util/log/eventpb";
+
+import "gogoproto/gogo.proto";
+import "util/log/logpb/event.proto";
+
+// Category: Contention events
+// Channel: SQL_EXEC
+//
+// Aggregated information about contention events.
+
+// Notes to CockroachDB maintainers: refer to doc.go at the package
+// level for more details. Beware that JSON compatibility rules apply
+// here, not protobuf.
+// *Really look at doc.go before modifying this file.*
+
+
+// AggregatedContentionInfo is recorded periodically when contention events
+// are resolved.
+message AggregatedContentionInfo {
+  CommonEventDetails common = 1 [(gogoproto.nullable) = false, (gogoproto.jsontag) = "", (gogoproto.embed) = true];
+  string waiting_stmt_fingerprint_id = 2  [(gogoproto.jsontag) = ",omitempty"];
+  string waiting_txn_fingerprint_id = 3 [(gogoproto.jsontag) = ",omitempty"];
+  string blocking_txn_fingerprint_id = 4 [(gogoproto.jsontag) = ",omitempty"];
+  string contended_key = 5 [(gogoproto.jsontag) = ",omitempty"];
```

Comment:
These are going to get marked as sensitive because they're strings, you could make them use the RedactableString type in the struct to enable partial redaction. Take a look at `job_events.proto` for instance, the `description` field in `StatusChange` does this.

Reply:
Gotcha, changed the fingerprint IDs and key to be partially sensitive/redacted